client/ionic/.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
We have multiple gitignores - just use one?

Reply:
nvm, it appears this structure relies on it.

Reply:
They can get pretty hairy and specific to the sub-projects. It's sometimes a little cleaner to target them where needed.  But perhaps Sam intends to merge them.